test/constants.js

```diff
@@ -0,0 +1,23 @@
+const constants = {
+	time: {
+		IMPRESSION_THRESHOLD : 100,
+		RAF_THRESHOLD : 10,
```

Comment:
This should probably be at least 16ms since that's the length of a frame under normal circumstances